.github/workflows/node.js.yml

```diff
@@ -127,7 +127,8 @@ jobs:
       - uses: pulumi/actions@v3
         with:
           command: up
-          stack-name: dev
+          stack-name: prod
+          upsert: true
```

Comment:
`upsert` will create the stack that is currently missing.

Reply:
That will do the trick!